.github/workflows/test-templates.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         go-version:
           - 1.21.x
         node-version:
-          - 18.x
+          - 22.x
```

Comment:
We should probably update to 20.x version here.  We support any node versions that are not EOL yet.  It's probably best to use the oldest supported version here, as it's more likely we break that, than a newer version.

Reply:
Thanks. Would you mind if I add to the test matrix `20` and `22`?

Reply:
Having both sounds fine to me.

Reply:
22 is LTS but we generally only test against oldest supported for these tests

Reply:
I added both and a link to where to find NodeJS release schedule.

Reply:
Updated to only use NodeJS 20.